components/google_drive/google_drive.app.js

```diff
@@ -26,11 +31,51 @@ module.exports = {
         return this._listDriveOptions(nextPageToken);
       },
     },
+    folderId: {
+      type: "string",
+      label: "Folder",
+      description: "The folder in the drive",
+      options({
+        prevContext,
+        drive,
+        baseOpts = {
+          q: `mimeType = '${GOOGLE_DRIVE_FOLDER_MIME_TYPE}'`,
+        },
+      }) {
+        const { nextPageToken } = prevContext;
+        return this.listDriveFilesOptions(drive, nextPageToken, baseOpts);
+      },
+    },
+    fileId: {
+      type: "string",
+      label: "File",
+      description: "The file in the drive",
+      options({
+        prevContext,
+        drive,
+        baseOpts = {
+          q: `mimeType != '${GOOGLE_DRIVE_FOLDER_MIME_TYPE}'`,
+        },
+      }) {
+        const { nextPageToken } = prevContext;
+        return this.listDriveFilesOptions(drive, nextPageToken, baseOpts);
```

Comment:
Can you add the path file? I think in some cases, the user may have the same file name in different folders. Show the file may help it

Reply:
I added file paths to labels for files and folders listed in async options for props used in actions.

For example, if the file path is `MyDrive/folder1/folder2/myFile.csv`, the option label would be `folder1 > folder2 > myFile.csv`, using angle brackets similar to the Google Drive UI. If the file path is `MyDrive/folder1/folder2/folder3/folder4/myFile.csv`, the option label would be `... > folder4 > myFile.csv`, truncating the file path so the file name is displayed within the UI.